### PR TITLE
Get orders history

### PR DESCRIPTION
**Pull request by luizakp**
Example by user:
image
Example by orders:
image

### CODE REVIEW

**Comment by luizakp** [comment]
Missing: index failed decoding orders

File: src/ComposableCow.ts [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+    user = await context.db.User.create({
+      id: userId,
+      data: {
+        address: event.args.owner,
+        chainId: context.network.chainId,
+      },
+    });
+  }
+
+  //decode stop loss static input
```
Comment (yvesfracari) [inline src/ComposableCow.ts]: it would be nice to have the stop loss contract here

File: src/ComposableCow.ts [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+        buyTokenPriceOracle: stopLossData[10],
+        strike: stopLossData[11],
+        maxTimeSinceLastOracleUpdate: stopLossData[12],
+        orderContract: event.args.params.handler,
+        user: user.id,
+      },
+    });
+  } catch (e) {
+    return;
+  }
```
Comment (yvesfracari) [inline src/ComposableCow.ts]: With this implementation we're not indexing the errors, right? IMO we should do it to later on show them on the frontend as well

**Review by yvesfracari — CHANGES_REQUESTED** [review]

File: src/ComposableCow.ts (hunk 2 above)
Comment (luizakp) [reply src/ComposableCow.ts]: yup, I have another PR for that: https://github.com/bleu-fi/composable-cow-api/pull/2

File: src/ComposableCow.ts (hunk 1 above)
Comment (luizakp) [reply src/ComposableCow.ts]: what do you mean?

Comment (yvesfracari) [reply src/ComposableCow.ts]: the comment should mention the link / address for the stop loss contract.

**Review by yvesfracari — APPROVED** [review]

**Pull request merged**